src/seedu/addressbook/AddressBook.java

```diff
@@ -374,9 +410,9 @@ private static String executeCommand(String userInputString) {
         case COMMAND_FIND_WORD:
             return executeFindPersons(commandArgs);
         case COMMAND_LIST_WORD:
-            return executeListAllPersonsInAddressBook();
+            return listAllPersonsInAddressBook();
         case COMMAND_DELETE_WORD:
-            return executeDeletePerson(commandArgs);
+            return deletePerson(commandArgs);
```

Comment:
Probably arguable :-). Your version is more concise (shorter), the original helps to place all "execution" methods closeby in listing as they all start with the format "executeXXXX"